Package.swift

```diff
@@ -24,6 +24,11 @@
 
 import PackageDescription
 
+// To use a local installation of ArcGIS Runtime for iOS, uncomment the following lines and also uncomment
+// package depenencies:package:path and comment the depenencies:package:url
+// import class Foundation.ProcessInfo
```

Comment:
I'm wondering if we should even keep these commented out for an even easier ease of use? The variables will be unused but it may not really add that much overhead. Thoughts?

Reply:
benefits of leaving them commented:
- the variable isn't used, so it's uncommented existence can be confusing
- if for some reason the force unwrap fails on some machine then leaving it commented won't cause any problems